packages/venia-concept/src/components/SignIn/signIn.js

```diff
@@ -0,0 +1,93 @@
+import { createElement, Component } from 'react';
+import PropTypes from 'prop-types';
+import Input from 'src/components/Input';
+import Button from 'src/components/Button';
+import defaultClasses from './signIn.css';
+import classify from 'src/classify';
+import Form from 'src/components/Form';
+import ErrorDisplay from 'src/components/ErrorDisplay';
+
+class SignIn extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            signInSection: PropTypes.string,
+            signInDivider: PropTypes.string,
+            forgotPassword: PropTypes.string,
+            root: PropTypes.string,
+            signInError: PropTypes.string,
+            showCreateAccountButton: PropTypes.string
+        }),
+
+        signInError: PropTypes.object,
+        signIn: PropTypes.func
+    };
+
+    state = {
+        password: '',
+        username: ''
+    };
+
+    get errorMessage() {
+        const { signInError } = this.props;
+        return <ErrorDisplay error={signInError} />;
+    }
+
+    render() {
+        const { classes, showCreateAccountForm } = this.props;
+        const { onSignIn, errorMessage } = this;
+
+        return (
+            <div className={classes.root}>
+                <Form submitForm={onSignIn}>
+                    <Input
+                        onChange={this.updateUsername}
+                        helpText={'example help text'}
+                        label={'Username or Email'}
```

Comment:
Suggestion: If the user has filled out this field and then clicks "Create Account" instead, it should copy this value into the CreateAccount counterpart field.